### PR TITLE
Neural Network (Backpropagation)

### PR DESCRIPTION
**Pull request by abkds**
R script to apply neural networks on the
dataset.

### CODE REVIEW

**Comment by samyak-268** [comment]
What does the 3rd column in the results signify?

**Comment by abkds** [comment]
3rd column is a value between 0 and 1. It is the output of the sigmoid logistic function. So in case of two class classification if it is a value above 0.5 that means class "1" , otherwise class "0"

**Comment by samyak-268** [comment]
:+1:
